src/layouts/AboutLayout.js

```diff
@@ -0,0 +1,338 @@
+import React from "react";
+import styled from "styled-components";
+import Link from "next/link";
+
+const info = [
+  {
+    icon: "fa-plus",
+    title: "Share your actions",
+    text:
+      "Have you ever wondered what is being done about climate change in your region and globally? By enabling climate protectors to share their project on Climate Connect, we will provide a comprehensive up to date list of what is being done about climate change."
+  },
+  {
+    icon: "fa-chart-bar",
+    title: "Measure the impact",
+    text:
+      "You are thinking about getting involved in climate action, but don't know how you can be most effective? On Climate Connect each action will have an impact score. This way you can find out how you can maximize your impact"
+  },
+  {
+    icon: "fa-users",
+    title: "Collaborate globally",
+    text:
+      "Climate change is a global issue. This is why we want to create a global network of climate protectors. Climate Connect allows you to see where you can make the biggest difference with your personal strengths and skillset. If you're already working on an impactful project, you will be able to find collaborators here."
+  },
+  {
+    icon: "fa-lightbulb",
+    title: "Get inspired",
+    text:
+      "On Climate Connect, you will find blue prints of high-impact projects, that you will be able to replicate at your location. This way you can very easily find the most effective thing you can work on with your specific skillset."
+  },
+  {
+    icon: "fa-eye",
+    title: "Maximum transparency",
+    text:
+      "Our code is available open source and can be found in our Github repository. The impact algorithm will be open source and the community will make the decisions on how to develop it further."
+  }
+];
+
+const members = [
+  {
+    name: "Christoph Stoll",
+    image: "christophstoll.png",
+    location: "Erlangen, Germany"
+  },
+  {
+    name: "Thomas Bove",
+    image: "thomasbove.png",
+    location: "Paris, France"
+  },
+  {
+    name: "Tobias Rehm",
+    image: "tobiasrehm.png",
+    location: "Erlangen, Germany"
+  },
+  {
+    name: "Reece Langerock",
+    image: "reecelangerock.png",
+    location: "Chicago, USA"
+  },
+  {
+    name: "Michael Fischer",
+    image: "michaelfischer.png",
+    location: "Erlangen, Germany"
+  }
+];
+
+function renderBubbles(info) {
+  return info.map(i => {
+    return <InfoBubble data={i} key={i.title}></InfoBubble>;
+  });
+}
+
+function renderMemberInfo(members) {
+  return members.map(m => {
+    return <Member data={m} key={m.name}></Member>;
+  });
+}
+
+export default function about() {
+  return (
+    <Container>
+      <AboutImageContainer>
+        <HeadlineContainer>
+          <Headline>
+            <HeadlineText>CLIMATE CONNECT</HeadlineText>
+          </Headline>
+        </HeadlineContainer>
+      </AboutImageContainer>
```

Comment:
Do we really need 4 custom divs here? Seems like it could probably just be 1 or 2.

Reply:
I was able to remove one div, didn't find a way to remove more custom divs without losing vertical and horizontal centering as well as the box around the text.